server/views/components/form-submit-button/template.njk

```diff
@@ -1,4 +1,4 @@
-<div class='govuk-!-margin-top-6'>
+<div class='govuk-!-margin-top-{{ params.marginTop if params.marginTop else "6" }}'>
```

Comment:
There's a possibility this is necessary due to the current state of our components but I don't like that we allowing any number to be passed here. Button spacing should be consistent across the site.

Was there a reason this needed to be changed?